docs/components/google/google-account-picker.md

```diff
@@ -0,0 +1,38 @@
+# Google Account Picker
+
+## Overview
+Google Account Picker is a non-visible component which can be used to pick a Google (Gmail) account registered in the user's device. This component can be used to create a login/register like system in the app.
+<br><br>
+![Google Account Picker](/docs/assets/images/google/preview.gif)

```

Comment:
Path to image is wrong. When deployed, the /docs is removed, so consequently the image won't be shown
Remove all references to `/docs` from it

```suggestion
![Google Account Picker](/docs/assets/images/google/preview.gif)
```

Reply:
Done, changed the path to `/assets/images/google/google-account-picker`.

Reply:
@barreeeiroo ...